save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/basic/TestResourcesSelection.kt

```diff
@@ -201,6 +212,33 @@ fun testResourcesSelection(
                         setSelectedLanguageForStandardTests
                     )) {}
 
+                    div("input-group-prepend") {
+                        input(type = InputType.text, name = "itemText") {
+                            key = "itemText"
+                            attrs.set("class", "form-control")
+                            attrs {
+                                value = props.execCmd
+                                placeholder = "Execution command:"
+                                onChangeFunction = {
+                                    setExecCmd(it)
+                                }
+                            }
+                        }
+                    }
+                    div("input-group-prepend") {
+                        input(type = InputType.text, name = "itemText") {
+                            key = "itemText"
+                            attrs.set("class", "form-control")
+                            attrs {
+                                value = props.batchSizeForAnalyzer
+                                placeholder = "Batch size (one by default)"
```

Comment:
```suggestion
                                placeholder = "Batch size (default: 1)"
```
I think numeric symbol is more suitable here

Reply:
Changed